_data/vedic-partap/vedic-partap.md

```diff
@@ -0,0 +1,7 @@
+---
+name: Vedic Partap # No longer than 18 characters
+institution: Indian Institute of Technology Kharagpur # no longer than 58 characters
+profile_pic: vedic.jpg # Relative path to the image you uploaded
+quote: "Thanks to all my friends that were there when I needed them the most. They made me a better person. KGP ka tempo high hai !!!" # No longer than 100 characters

```

Comment:
Just noticed this, the quote character count limit exceeds the limit-100. Would you mind modifying this as well?